src/build/firefox.manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-	"version": "1.2.45_Firefox",
+	"version": "1.2.46",
```

Comment:
actually, FF supports such format, just that Chrome doesn't support. if we leave it as "1.2.46_Firefox" - we'll be able to track

Reply:
I think using `version_name` in both, and then using a consistent method to turn that in to a string we send in the rawlog body would be neater. It'd be good info, and I'm happy to pick that up in another PR, perhaps when I pick up the error logging task.

Reply:
ok